packages/app/utilities.ts

```diff
@@ -171,8 +171,10 @@ export const overrideMagicInstance = (email: string) => {
 
 // Format big numbers
 export function formatNumber(number: number) {
-  if (number > 1000000) {
-    return `${(number / 1000000).toFixed(1)}m`;
+  if (!number) return 0;
+  // for the edge case of 100k, our max supply, put “100k”, no decimals
+  if (number >= 100000) {
```

Comment:
We also use this function for likes and comments, even if this is not the case it might introduce a bug later on

Reply:
yeah my bad, will add a new method to use for claims.